apps/testing-javascript/.env.development

```diff
@@ -28,3 +28,8 @@ NEXT_PUBLIC_CONVERTKIT_TOKEN=CK_TOKEN
 NEXT_PUBLIC_CONVERTKIT_ANSWER_URL=/api/skill/answer/convertkit
 NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL=/api/skill/subscribe/convertkit
 NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL=/api/skill/subscriber/convertkit
+
+# Sanity
+NEXT_PUBLIC_SANITY_PROJECT_ID='p9j29730'
+NEXT_PUBLIC_SANITY_DATASET=production
```

Comment:
this corresponds to the Sanity project I created and seeded under the Testing JavaScript organization